app/templates/home.hbs

```diff
@@ -33,14 +33,9 @@
         <div class="sidebar">
           <p>Popular Tags</p>
           <div class="tag-list">
-            <a href="" class="tag-pill tag-default">programming</a>
-            <a href="" class="tag-pill tag-default">javascript</a>
-            <a href="" class="tag-pill tag-default">emberjs</a>
-            <a href="" class="tag-pill tag-default">angularjs</a>
-            <a href="" class="tag-pill tag-default">react</a>
-            <a href="" class="tag-pill tag-default">mean</a>
-            <a href="" class="tag-pill tag-default">node</a>
-            <a href="" class="tag-pill tag-default">rails</a>
+            {{#each model.popularTags as |tag|}}
```

Comment:
I would advise against `setupController` it's a bit of a noobie footgun and routes are not easy o test. if you don't want these on model aliasing them in the controller is safer and easier to test.